.github/workflows/tests.yml

```diff
@@ -129,8 +129,8 @@ jobs:
     - run: node lib/cli/cli install-deps
     - run: bash packages/installation-tests/installation-tests.sh
 
-  headful_linux:
```

Comment:
Let's keep the original values for internal systems - easier to keep required bots assigned